src/checkers/check_hr.py

```diff
@@ -135,9 +135,9 @@ def check_specific(filename: str, file: int, config_filename: str) -> int:
     answers_number_ok = False
     answers_ratio_ok = False
     for total_answers in answers_options.items():
-        if (correct_answers + wrong_answers) == total_answers:
+        if (correct_answers + wrong_answers) == total_answers[0]:
             answers_number_ok = True
-            if correct_answers == answers_options[total_answers]:
+            if correct_answers == total_answers[1]:
                 answers_ratio_ok = True
             break
 
```

Comment:
I think we should take advantage that answers_options is a dictionary and we can easily search it.
```suggestion
    supposed_correct_answers = answer_options[correct_answers + wrong_answers]
    if supposed_correct_answers == None:
        print_error(filename, line_idx,
                    f": The question does not have the right number of answers! (should be one of {", ".join(answer_options.keys())})\n")
    elif supposed_correct_answers != correct_answers:
        print_error(filename, line_idx,
                    ": The question has bad correct / wrong answers ratio!\n")
```